File: js/task2/S2_FOR_01.js
// S2_FOR_01

function fizzBuzz(n) {
    for (let i = 1; i <= n; i++) {
        let output = "";
        if (i % 3 === 0) output += "Fizz";
        if (i % 5 === 0) output += "Buzz";
        
        // if empty, just number
        if (output === "") {
            console.log(i);
        } else {
            console.log(output);
        }
    }
}

fizzBuzz(15);

File: python/task2b/S2_FOR_01.py
# S2_FOR_01

for i in range(1, 31):
    out = ""
    if i % 3 == 0: out += "Fizz"
    if i % 5 == 0: out += "Buzz"
    
    if out == "":
        print(i)
    else:
        print(out)